test/iot-agent/services

```diff
@@ -0,0 +1,215 @@
+#!/bin/bash
+#
+#  Command Line Interface to start all services associated with the Getting-Started Tutorial
+#
+#  For this tutorial the commands are merely a convenience script to run docker-compose
+#
+
+set -e
+
+SCORPIO="http://scorpio:9090/scorpio/v1/info/"
+ORION="http://orion:1026/version"
+IOT_AGENT="http://iot-agent:4041/version"
+CONTEXT="https://fiware.github.io/data-models/context.jsonld"
+
+if (( $# != 1 )); then
+	echo "Illegal number of parameters"
+	echo "usage: services [create|orion|scorpio|stop]"
```

Comment:
Need to update the tutorial, tell users that Scorpio is not supported till now for IOT agent integration.